tests/EpisodeTest.php

```diff
@@ -4,36 +4,311 @@
+
+    /**
+     * Assert that EpisodeRepository's updateEpisode updates an existing episode
+     * @return [type] [description]
+     */
+    public function testUpdateEpisode()
+    {
+        $repository = new EpisodeRepository();
+        $episode = $repository->createEpisode([
+            'episode_name' => 'Swanky new Episode',
+            'episode_description' => 'Swanky New episode description',
+            'view_count'            => 10,
+            'image'                 => "http://goo.gl/8sorZR",
+            'audio_mp3'             => "http://goo.gl/LkNP5M",
+            'channel_id'            => 1,
+            'status'                => 0,
+            'likes'                 => 10
         ]);
+        $update = $repository->updateEpisode($episode['id'], 'episode_name', 'Swanky updated name');
+        $this->assertTrue($update);
+
+        $getEpisode = $repository->findEpisodeById($episode['id'])->toArray();
+
+        $this->assertTrue(is_array($getEpisode));
+        $this->assertArrayHasKey('episode_name', $getEpisode);
+        $this->assertArrayHasKey('episode_description', $getEpisode);
+        $this->seeInDatabase('episodes', ['episode_name' => 'Swanky updated name']);
+    }
+    /**
+    * Assert that EpisodeRepository's
+    */
+    public function testActiveAndPendingEpisodes()
+    {
+        factory('Suyabay\Episode', 1)->create(['status' => 1]);
+        factory('Suyabay\Episode', 1)->create(['status' => 0]);
+
+        $repository = new EpisodeRepository();
+        $this->assertTrue(is_array($repository->getActiveEpisodes()->toArray()));
+        $this->assertTrue(is_array($repository->getPendingEpisodes()->toArray()));
+    }
+
+   /**
+    * Assert that singleEpisode method displays a single episode.
+    *
+    * @return void
+    */
+    public function testSingleEpisodeMethod()
+    {
+        factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel', 1)->create();
+        $episode = factory('Suyabay\Episode')->create(['status' => 1]);
+
+        $this->call(
+            'GET',
+            '/episodes/' . $episode['id']
+        );
+        $this->see($episode['episode_name']);
+
+        $this->assertViewHasAll(['episodes', 'channels']);
+        $this->seeInDatabase('episodes', ['episode_name' => $episode['episode_name']]);
+    }
+
+    /**
+     * Assert that admin user can see the dashboard stats.
+     *
+     * @return [type] [description]
+     */
+    public function testAdmincanSeeStats()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard'
+            );
+        $this->assertViewHas('data');
+        $this->see($episodes[0]['episode_name']);
+        $this->see($episodes[0]['episode_description']);
+        $this->see(3);
```

Comment:
**Code Quality - Critical**

Please replace this with a proper implementation of asserting that the admin can actually see the total number of users registered on the platform